<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewtonRootColoring</class>
 <widget class="QWidget" name="NewtonRootColoring">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>375</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>280</width>
    <height>375</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>280</width>
    <height>375</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Newton Root Coloring</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>110</height>
    </rect>
   </property>
   <property name="title">
    <string>First Root Area</string>
   </property>
   <widget class="QLineEdit" name="firstRootRedLineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="firstRootGreenLineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="firstRootBlueLineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>22</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Red :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>52</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Green :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>82</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Blue :</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>261</width>
     <height>110</height>
    </rect>
   </property>
   <property name="title">
    <string>Second Root Area</string>
   </property>
   <widget class="QLineEdit" name="secondRootBlueLineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>82</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Blue :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>22</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Red :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="secondRootRedLineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>52</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Green :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="secondRootGreenLineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>196</x>
     <y>345</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>261</width>
     <height>110</height>
    </rect>
   </property>
   <property name="title">
    <string>Second Root Area</string>
   </property>
   <widget class="QLineEdit" name="thirdRootBlueLineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="thirdRootRedLineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>52</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Green :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>22</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Red :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="thirdRootGreenLineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>82</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Blue :</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>okButton</tabstop>
  <tabstop>firstRootRedLineEdit</tabstop>
  <tabstop>firstRootGreenLineEdit</tabstop>
  <tabstop>firstRootBlueLineEdit</tabstop>
  <tabstop>secondRootRedLineEdit</tabstop>
  <tabstop>secondRootGreenLineEdit</tabstop>
  <tabstop>secondRootBlueLineEdit</tabstop>
  <tabstop>thirdRootRedLineEdit</tabstop>
  <tabstop>thirdRootGreenLineEdit</tabstop>
  <tabstop>thirdRootBlueLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
